products/rhel8/profiles/stig.profile

```diff
@@ -1173,3 +1173,6 @@ selections:
 
     # RHEL-08-040390
     - package_tuned_removed
+
+    # RHEL-08-010163
+    - package_krb5-server_removed
```

Comment:
Thank you very much for enhancing the RHEL8 profile as well.
We have some additional tests for RHEL profiles to ensure that they do not get modified by accident. We store current compiled version of the profile (without comments etc) in tests/data/profile_stability/rhel8/stig.profile.
Could you please do the following:
1. ./build_product rhel8
2. cp build/rhel8/profiles/stig.profile tests/data/profile_stability/rhel8/stig.profile
3. the same for stig_gui.profile cause it extends the stig profile
4. commit

Thank you.